app/src/main/java/com/example/musicapp/view/StringUtils.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.mvpmusicapp.ui
+
+import android.content.Context
+import com.example.musicapp.R
+
+
+fun convertToDurationFormat(context: Context, duration: Double?): String {
+    val durationStr = String.format(
+        context.getString(R.string.text_format_duration),
+        (duration?.div(60000))
+    ).replace(
+        if (duration.toString().contains(context.getString(R.string.text_comma))) context.getString(
+            R.string.text_comma
+        )
+        else context.getString(R.string.text_dot), context.getString(R.string.text_colon)
+    )
+
+    val partOfTime = durationStr.split(context.getString(R.string.text_colon))
+    val durationBuilder =
+        StringBuilder(partOfTime[0]).append(context.getString(R.string.text_colon))
+    if (partOfTime.size > 1) {
+        if (((partOfTime[1].toDouble() / 100) * 60).toInt() < 10) durationBuilder.append(
```

Comment:
với những logic tính toán khá phức tạp như đoạn if của ông ntn thì ông có thể tạo 1 Extension Function để sau gọi lại cho tiện nhé